### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by Gara64**
* Do not open a new tab after installation of the extension
* Use a sad cozy icon if no cipher was found
* The message toast was overlapping with the tab menu: we move it up
* Use "Cozy password" instead of "Master password"

### CODE REVIEW

File: src/popup/scss/misc.scss [hunk 1]
```diff
@@ -301,3 +301,14 @@ app-vault-icon {
 .draggable {
     cursor: move;
 }
+
+.toast-bottom {
+    bottom: 55px;
+    right: 10px;
+    left: 10px;
+    width: 100%;
+}
```
Comment (Crash--) [inline src/popup/scss/misc.scss]: Are we sure about this change? Does this modification is well scoped and can't have impact elsewhere?

Comment (Gara64) [reply src/popup/scss/misc.scss]: We tested with @y-lohse and it looked fine. Do you worry about something in particular?

Comment (Crash--) [reply src/popup/scss/misc.scss]: I don't know if the toaster comes with a prebuilt `toast-bottom` class and then you're overriding it (this why I'm worried) or if this is something you created ?

Comment (Gara64) [reply src/popup/scss/misc.scss]: Yes, this is something I created, and it is not overriding an existing class, the prebuilt ones are [here](https://github.com/Stabzs/Angular2-Toaster/#container-position) ;)

File: src/popup/scss/misc.scss [hunk 2]
```diff
@@ -301,3 +301,14 @@ app-vault-icon {
...
+
+#toast-container .toast.toast-info {
+    width: 96%;
```
Comment (Crash--) [inline src/popup/scss/misc.scss]: Where this magic number comes from? Can you express it with variables? I don't know how to do that with `sass` anymore but something like: 
const MARGIN_CONTAINER = 2%
width: 100% - 2*MARGIN_CONTAINER

Comment (Gara64) [reply src/popup/scss/misc.scss]: This is actually something I took from the original class: `https://github.com/Stabzs/Angular2-Toaster/blob/master/src/toaster.css#L223`. Maybe a reference to this line would be enough?

Comment (Crash--) [reply src/popup/scss/misc.scss]: Yep :) thanks!

**Pull request merged**